diktat-rules/src/test/resources/test/smoke/src/main/kotlin/Example8Expected.kt

```diff
@@ -0,0 +1,10 @@
+package org.cqfn.diktat
+
+/*
+ * Copyright (c) Your Company Name Here. 2010-2020
+ */
+
+fun foo() {
+    run { }
```

Comment:
Doesn't `EmptyBlock` trigger in this case?

Reply:
No. The smoke test works out correctly. Probably the rule doesn't work for lambda.

Reply:
Please check, I'm sure I have seen it trigger on lambdas before.